new-chapter.md

```diff
@@ -0,0 +1,37 @@
+# PyLadies Request to Start Chapter Form
+
+Please fill in the following form to the best of your knowledge, and keep in mind that PyLadies is managed on a volunteer basis and the approval or feedback on the application might take a while. We will contact you as soon as we can.
+
+## Organizer
+
+- `your name`
+- `what are your pronouns (e.g. she/her, they/them)?`
+- `your contact email`
+- `username on PyLadies slack (You can join at https://slackin.pyladies.com/.)`
```

Comment:
```suggestion
- `username on PyLadies slack (Register first at https://slackin.pyladies.com/.)`
```

In the previous form, I have seen examples where they entered an invalid Slack username (which broke the bot's automation). So I'm hoping that by making it more explicit that they need to first create the slack account, it could help us a bit. What do you think?